Directory.Build.props

```diff
@@ -44,21 +44,20 @@
 
     <PropertyGroup>
         <IncludeCompatabilityAnalyzer Condition=" '$(IncludeCompatabilityAnalyzer)' == '' ">True</IncludeCompatabilityAnalyzer>
+        <RunAnalyzersDuringBuild>true</RunAnalyzersDuringBuild>
+        <RunAnalyzersDuringLiveAnalysis>true</RunAnalyzersDuringLiveAnalysis>
+        <RunAnalyzers>true</RunAnalyzers>
```

Comment:
Did we have IncludeCompatibilityAnalyzer set to true before? I'm trying to understand this particular change and why you moved it here

Reply:
Yes; I simply moved these 3 properties into the IncludeCompatbilityAnalyzer's `PropertyGroup`, but I could put them into their own if desired for readability

Reply:
I moved them out of the `PropertyGroup` that only evaluates during debug

Reply:
I just killed them. Apparently they all default to `true`

Reply:
Sounds good